src/app/projects/new-project/new-project.component.ts

```diff
@@ -39,8 +42,9 @@ export class NewProjectComponent implements OnInit {
     this.projectsService.addProject(this.project)
       .subscribe(response => {
         this.project = response;
-        setTimeout(() => this.messageService.add({ severity: 'success', summary: 'Sukces', detail: 'Projekt utworzony!' }));
-        this.router.navigateByUrl('/projects');
+        this.storeService.setCurrentProject(this.project);
```

Comment:
Nie może to działać w taki sposób, że dodawany jest projekt, który nie jest w pełni skonfigurowany. F5 na widoku klucza (**a nawet powrót**) spowoduje, że już nie będzie się dało nigdy ustawić jiry, a projekt pojawi się na liście.

Reply:
To z jednej strony jest ok - wtedy konfiguracja mogłaby być w MENU też, a produkt by działał bez Jiry, tylko że produkt i tak teraz średnio będzie działał bez Jiry, bo policzenie współczynnika by wymagało dzielenia przez zero. Poza tym ten accessToken, który siedzi w bazie jest chyba ważny do pięciu lat, a admin może odejść z projektu w dowolnym momencie i nie wiem, co się wtedy dzieje

ALE pewnie lepiej teraz wymusić tę Jirę, bo nie ogarniemy na piątek przypadku bez Jiry raczej

Reply:
skoro już chcecie to zrobić jako taki 'kreator', to kolejne widoki powinny trzymać wspólny stan i dopiero ostatnie _akceptuj_ dodaje projekt a następnie ustawia jirę

Reply:
jak mi wytłumaczysz jak coś takiego zrobić, to z chęcią zrobię.

Reply:
Ja tego nie zrobię na backendzie jakby co

Reply:
Absolutnie nie

Reply:
To jest _kreator_, bo tam się różne niesamowite rzeczy dzieją pod spodem

Reply:
Mogę zrobić endpoint na GET boarda i jak nie ma, to przy wyborze znowu się trafia do _kreatora_

Reply:
w sumie też to z chęcią zrobię xd ale jak bardzo się uprzesz to może mnie przekonasz xd

Reply:
To było do mnie czy do @Numernabis?

Reply:
na razie jesteśmy dumni, że w ogóle działa. nie idealnie, ale działa.

Reply:
Jak będziesz hejtował, to usuniemy logo z widoczków

Reply:
Stworzymy WŁASNE

Reply:
Btw, nie ma papajodżiry na ekranie logowania :((((